lnwallet/channel.go

```diff
@@ -833,8 +833,7 @@ type LightningChannel struct {
 	// updates to this channel. The log is walked backwards as HTLC updates
 	// are applied in order to re-construct a commitment transaction from a
 	// commitment. The log is compacted once a revocation is received.
-	localUpdateLog  *updateLog
-	remoteUpdateLog *updateLog
+	updateLogs lntypes.Dual[*updateLog]
```

Comment:
I don't think there's a way around the symmetric design. It's the nature of the problem. I just think we need to make sure it's clear what each one represents. This can be accomplished with godoc etc.